### PR TITLE
Steam Test Server - Update 23

### PR DESCRIPTION
**Pull request by Sunaries**
- Defined a few of the current map names in Lithuanian with the help of @Xenobius2501 and @lordseph  ('Our Lithuanian guys'). Thank you so much. 
- Fixed Ukert's NPC title.
- Standardized NPC titles of the Royal Army guards.
- Improved sentences.
- Changed Magic Scroll to Enchant Scroll.
- Fixed 'herb' to 'Thunderous Herb'.
- Changed the Jolly Roger skill description. Thanks @sususuyo !
- Added contributions of @TentuZero. Thank you very much.
- Added ITEM contributions of Kei and Kanra. Thanks.

### CODE REVIEW

File: EnglishTranslation-master/SKILL.tsv [hunk 1]
```diff
@@ -5744,7 +5744,7 @@ SKILL_20151224_005743	Increases attack equal to [Skill Level+1]x[Number of Weapo
 SKILL_20151224_005744	Penalty Reduction
 SKILL_20151224_005745	Reduces the penalty from incapable of combat.
 SKILL_20151224_005746	Equipment Durability Penalty: -#{CaptionRatio}#%{nl}Consumed Silver Penalty: -#{CaptionRatio2}#%{nl}Consumed Gem Penalty: -#{CaptionTime}#%
-SKILL_20151224_005747	Plant a flag that raises the combat abilities of the pirate crew. A combo is activated if multiple enemies is defeated around the jolly roger. And at a certain chance, attacking enemies allows you to pillage items and the higher your Jolly Roger level is, the higher the chance. (Combo is only activated if the party leader is a Corsair.)
+SKILL_20151224_005747	Place a flag that raises the combat abilities of the pirate crew. A combo is activated if multiple enemies is defeated around the jolly roger. And at a certain chance, attacking enemies allows you to pillage items and the higher your Jolly Roger level is, the higher the chance. (Combo is only activated if the party leader is a Corsair.)
```
Comment (sususuyo) [inline EnglishTranslation-master/SKILL.tsv]: `And at a certain chance, attacking enemies allows you to pillage items and the higher your Jolly Roger level is, the higher the chance.`
maybe we should rework this to
`Attacking enemies within the jolly roger allows you to pillage items at a chance based on your Jolly Roger level.`

Also, does every instance of "jolly roger" need to be capitalized in this line? if so, you should probably do that

File: EnglishTranslation-master/SKILL.tsv [hunk 2]
```diff
@@ -2033,11 +2033,11 @@ SKILL_20150317_002032	Strike
 SKILL_20150317_002033	Already knocked back enemy is knocked back again and receives additional damage.
 SKILL_20150317_002034	Nanta
 SKILL_20150317_002035	Reduce Knockback range of monsters when using whirlwind.
-SKILL_20150317_002036	STR increases for Melee-type attacks.
-SKILL_20150317_002037	Agility is increased. Affects ranged Physical attacks, chance of critical, and evasion.
-SKILL_20150317_002038	Physical strength improves. Max HP increases.
+SKILL_20150317_002036	Increased STR. Affects your melee physical attacks.
+SKILL_20150317_002037	Increased DEX. Affects your ranged physical attacks, critical rate and evasion.
+SKILL_20150317_002038	Increased CON. Affects your maximum HP and stamina.
```
Comment (Sourpusss) [inline EnglishTranslation-master/SKILL.tsv]: There's an extra space after the period.  I'm adding this line to my PR, but without the space.

File: EnglishTranslation-master/SKILL.tsv [hunk 3]
```diff
@@ -4433,7 +4433,7 @@ SKILL_20150714_004432	Enemies frozen by a Cryomancer will receive 10% additional
 SKILL_20150714_004433	Increases the damage dealt on an enemy with [Telekinesis] by 1% per attribute level.
 SKILL_20150714_004434	Increases the damage dealt on an enemy with [Psychic Pressure] by 1% per attribute level.
 SKILL_20150714_004435	Increases the damage dealt on an enemy with [Magnetic Force] by 1% per attribute level.
-SKILL_20150714_004436	Enemies hit by [Psychic Pressure] have a 10% chance per attribute level to become afflicted with [Stun]. for 1 second.
+SKILL_20150714_004436	Enemies hit by [Psychic Pressure] have a 10% chance per attribute level to become afflicted with [Stun] for 1 second.
```
Comment (Sourpusss) [inline EnglishTranslation-master/SKILL.tsv]: Extra space at the end.  That explains my conflict!

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ITEM.tsv [hunk 4]
```diff
@@ -9268,11 +9268,11 @@ ITEM_20151224_009267	Bead Amulet
 ITEM_20151224_009268	Cross Medal Amulet
 ITEM_20151224_009269	Gem Amulet
 ITEM_20151224_009270	Dilgele Seeds
-ITEM_20151224_009271	You can plant these seeds to grow produce. {nl}They can be used only within your guild's hideout.
+ITEM_20151224_009271	You can plant these seeds to grow crops.{nl}Can only be used in a Guild Hangout.
 ITEM_20151224_009272	Gyslotis Seeds
 ITEM_20151224_009273	Raudonelis
 ITEM_20151224_009274	Guinea Pig Egg
-ITEM_20151224_009275	You can grow a companion from this egg. {nl}It can be used only within your guild's hangout.
+ITEM_20151224_009275	You can grow a companion from this egg. {nl}It can only be used in a Guild Hangout.
```
Comment (grillo-delmal) [inline EnglishTranslation-master/ITEM.tsv]: Wouldn't it be better if both lines 9271 and 9275 had the same sentence in the end?

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**